frontend/packages/dev-console/src/components/helm/__tests__/HelmReleaseResourceTableRow.spec.tsx

```diff
@@ -37,7 +37,7 @@ describe('HelmReleaseResourceTableRow', () => {
       helmReleaseResourcesTableRow
         .find(TableRow)
         .childAt(1)
-        .hasClass(tableColumnClasses.kind),
```

Comment:
if you change to an indexed className, you can deploy a loop here and reduce code.
Also, you can check for other properties like the text of the tableHeaderRow(if applicable)

Reply:
This entire tests isn't useful. It asserts the presence of static classes that are always present. There is no conditional code being tested. This is simply adding a test for the sake of adding a test and will required updating every time we change the static rendering result.

Reply:
I have removed the unwanted tests from this file and have added a test to check for the values rendered under the Status column